homeassistant/components/ihc/sensor.py

```diff
@@ -55,7 +62,7 @@ def __init__(
         self._unit_of_measurement = unit
 
     @property
-    def device_class(self):
+    def device_class(self) -> SensorDeviceClass | None:
```

Comment:
This is type hint improvement - unrelated to config-flow. Please move to preliminary/follow-up PR.

Reply:
Done in PR [90350](https://github.com/home-assistant/core/pull/90350)